packages/core/src/payment/strategies/amazon-pay-v2/amazon-pay-v2-payment-processor.ts

```diff
@@ -109,7 +107,9 @@ export default class AmazonPayV2PaymentProcessor {
         };
 
         if (features['PROJECT-3483.amazon_pay_ph4']) {
-            return /^(SANDBOX|LIVE)/.test(publicKeyId)
+            const amount = getCheckout()?.outstandingBalance.toString();
```

Comment:
```suggestion
            const amount = getCheckoutOrThrow()?.outstandingBalance.toString();
```

Reply:
Since it's not a strictly required parameter for Amazon, I don't want it to be a blocker to checkout.